lib/screens/IconExplanationsScreen.dart

```diff
@@ -0,0 +1,182 @@
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: [
+                    Image.asset(
+                      'assets/icons/youth_club_fett.png',
+                      height: 30,
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(20),
+                      child: Text(
+                        'Community Youth Club (CTC)',
+                        style: TextStyle(
+                          fontSize: 20,
+                          //fontWeight: FontWeight.bold
+                        ),
+                      ),
+                    )
+                 ],
+                )
+            ),
+            Padding(
+                padding: const EdgeInsets.only(left: 30),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: [
+                    Image.asset(
+                      'assets/icons/phonecall_pe_fett.png',
+                      height: 30,
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(20),
+                      child: Text(
+                        'Phone Call by PE',
+                        style: TextStyle(
+                            fontSize: 20,
+                        ),
+                      ),
+                    )
+                  ],
+                )
+            ),
+            Padding(
+                padding: const EdgeInsets.only(left: 30),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: [
+                    Image.asset(
+                      'assets/icons/homevisit_pe_fett.png',
+                      height: 30,
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(20),
+                      child: Text(
+                        'Home Visit by PE',
+                        style: TextStyle(
+                          fontSize: 20,
+                        ),
+                      ),
+                    )
+                  ],
+                )
+            ),
+            Padding(
+                padding: const EdgeInsets.only(left: 30),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: [
+                    Image.asset(
+                      'assets/icons/nurse_clinic_fett.png',
+                      height: 30,
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(20),
+                      child: Text(
+                        'By the nurse at the clinic',
+                        style: TextStyle(
+                          fontSize: 20,
+                        ),
+                      ),
+                    )
+                  ],
+                )
+            ),
+            Padding(
```

Comment:
If there is exactly the same code over and over again I usually write a function that does it. This will help reduce code duplication. E.g.

```
Padding _makeExplanation(String iconAsset, String explanation) {
  return Padding(
      padding: const EdgeInsets.only(left: 30),
      child: Row(
        mainAxisAlignment: MainAxisAlignment.start,
        children: [
          Image.asset(
            iconAsset,
            height: 30,
          ),
          Padding(
            padding: const EdgeInsets.all(20),
            child: Text(
              explanation,
              style: TextStyle(
                fontSize: 20,
                //fontWeight: FontWeight.bold
              ),
            ),
          )
        ],
      )
  );
}
```

Then we can just call

```_makeExplanation('assets/icons/youth_club_fett.png', 'Community Youth Club (CTC)')```

Reply:
Ok, that is actually a very smart idea. I will refactor the code.